app/views/defence_requests/new.html.erb

```diff
@@ -82,6 +82,13 @@
 
 <script>
  $(document).ready(function () {
+
+  $(document).on('keydown', function(event) {
+    if($('.solicitor_results_list *').length > 0 && event.keyCode === 27){
```

Comment:
afaik ESC is consistent. If there is an issue with it there is still the Close link as a fallback
I'm going to look into something like browserstack for beta so we can be more certain about these sorts of things